File: snippets/ts/src/tour/workflows/serving_lambda.ts
const signupWorkflow = restate.workflow({
  name: "MyWorkflow",
  handlers: { run: async () => {} },
});

// <start_lambda>
import * as restate from "@restatedev/restate-sdk/lambda";
export const handler = restate
  .endpoint()
  .bind(signupWorkflow)
  .handler();
// <end_lambda>

File: snippets/go/tour/workflows/servinglambda.go
package workflows

import (
	"log"

	"github.com/aws/aws-lambda-go/lambda"
	restate "github.com/restatedev/sdk-go"
	server "github.com/restatedev/sdk-go/server"
)

type SignupWorkflow struct{}

func serveLambda() {
	// <start_here>
	handler, err := server.NewRestate().
		Bind(restate.Reflect(SignupWorkflow{})).
		Bidirectional(false).
		LambdaHandler()
	if err != nil {
		log.Fatal(err)
	}
	lambda.Start(handler)
	// <end_here>

}
